acc_test/resource_cluster_test.go

```diff
@@ -342,7 +342,7 @@ func TestAccOpenSearchClusterResize(t *testing.T) {
 					time.Sleep(1 * time.Minute)
 				},
 				Config:      invalidResizeConfig,
-				ExpectError: regexp.MustCompile("t3.small"),
+				ExpectError: regexp.MustCompile("SRH-DEV-t4g.small"),
```

Comment:
typo in `SRH-DEV-t4g.small` ?

Reply:
What typo? this is to match the invalid test case for opensearch cluster